### ISSUE TITLE
Remember view extent

### ISSUE THREAD

**Issue by skeenp** [DMS-Aus/Roam#308]
It would be nice for a roam project to optionally be able to remember the last extent, and reload that on project startup. Returning to home each time the program is reset can be inefficient.

**Comment by NathanW2** [comment]
Yeah this is planned for the next version.  Open to pull requests for it.
Would be easy to add if you want to tackle it.
